Sprint-3/2-practice-tdd/count.js

```diff
@@ -1,5 +1,21 @@
 function countChar(stringOfCharacters, findCharacter) {
-  return 5
+  if (
+    typeof stringOfCharacters !== "string" ||
+    typeof findCharacter !== "string"
+  ) {
+    return 0;
+  }
+  if (findCharacter.length !== 1) {
```

Comment:
This is a good check! As an extension, what if `stringOfCharacters.length` is 0 and `findCharacter.length` is 1? I see you now return just 0 but would it perhaps make more sense to throw an error because there is nothing to search character in?